<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addseance</class>
 <widget class="QWidget" name="addseance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white
</string>
  </property>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;
font: 700 28pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Planification de seance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Saisir l'horaire</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_2">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
      <item>
       <property name="text">
        <string>2h-4h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17h-19h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20h-22h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22h-00h</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Saisir la salle</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_3">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Saisir le prix</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_4">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
      <item>
       <property name="text">
        <string>30dhs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40dhs</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Saisir la capacit√©</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
color:black</string>
      </property>
      <property name="text">
       <string>Afficher les films</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
